package.json

```diff
@@ -57,6 +57,11 @@
       "name": "brodybits",
       "email": "chris@brody.consulting",
       "url": "https://github.com/brodybits"
+    },
+    {
+      "name": "Christian Bewernitz",
+      "email": "coder@karfau.de",
+      "web": "https://github.com/karfau"
```

Comment:
nit: This update seems orthogonal (unrelated) to me from actually changing the scope to use the @xmldom org.

I start to wonder if we should remove the maintainers & contributors entries when we make the next minor release (0.8.0). I have a feeling that this kind of info should be available from npm & GitHub, would be nice if we don't have to keep synchronization as maintainers come and go in the future.

Reply:
You are right, thx for pointing it out.
Reverted the related changes.
I like the idea of not synchronizing thisinformation into `package.json`.